packages/console/src/ConsoleInput.scss

```diff
@@ -74,3 +74,16 @@ $console-input-margin-color: $foreground;
     }
   }
 }
+
+.docs {
```

Comment:
That's a pretty greedy selector, and not scoped. Where is ".docs" as the class defined?

I think Editor.css is probably a better place for this as well.